src/main/java/blusunrize/immersiveengineering/common/IESaveData.java

```diff
@@ -46,7 +50,13 @@ public void readFromNBT(NBTTagCompound nbt)
 				}
 			}
 		}
-
+		
+		Minecraft.getMinecraft().mouseHelper.ungrabMouseCursor();

```

Comment:
From what I know right know that should be ok. Just making the metal press a bit more fancy right now (Accept `<minimum input per mold>` in the first take instead of 1, would make it behave less weird in most situations, like in Dire's video today).